.ibm/pipelines/openshift-ci-tests.sh

```diff
@@ -251,7 +308,7 @@ check_and_test() {
     echo "Backstage is not running. Exiting..."
     exit 1
   fi
-    run_tests $namespace
+    run_tests $release_name $namespace
```

Comment:
@zdrapela minor cosmetic issue :) - indentation may need to be fixed

Reply:
Thank you! Fixed 😉 Please add LGTM again after that